index.html

```diff
@@ -0,0 +1,140 @@
+        </ol>
+
+    </section>
+
+    <section id="Javascript_and_Java" class="main-section">
+      <header >
+        Javascript and Java
+      </header>
+      <p>JavaScript and Java are similar in some ways but fundamentally different in some others. The JavaScript
+        language resembles Java but does not have Java's static typing and strong type checking. JavaScript follows most
+        Java expression syntax, naming conventions and basic control-flow constructs which was the reason why it was
+        renamed from LiveScript to JavaScript. </p>
+      <p>In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime
+        system based on a small number of data types representing numeric, Boolean, and string values. JavaScript has a
+        prototype-based object model instead of the more common class-based object model. The prototype-based model
+        provides dynamic inheritance; that is, what is inherited can vary for individual objects. JavaScript also
+        supports functions without any special declarative requirements. Functions can be properties of objects,
+        executing as loosely typed methods. </p>
+      <p>JavaScript is a very free-form language compared to Java. You do not have to declare all variables, classes,
+        and methods. You do not have to be concerned with whether methods are public, private, or protected, and you do
+        not have to implement interfaces. Variables, parameters, and function return types are not explicitly typed.
+      </p>
+    </section>
+
+    <section id="Hello_World" class="main-section">
+      <header >
+        Hello World
+      </header>
+      <p>To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code:
+      </p>
+      <code>function greetMe(yourName) { alert("Hello " + yourName); }</code>
+      <code>greetMe("World");</code>
+
+      <p>Select the code in the pad and hit Ctrl+R to watch it unfold in your browser!</p>
+    </section>
+
+    <section id="Declaring_Variables" class="main-section">
+
+      <header >
+        Declaring Variables
+      </header>
+      <p>You can declare a variable in three ways:
+
+        With the keyword var. For example, </p>
+      <code>var x = 42.</code>
+      <p>This syntax can be used to declare both local and global variables.
+
+        By simply assigning it a value. For example, </p>
+      <code>x = 42.</code>
+      <p>This always declares a global variable. It generates a strict JavaScript warning. You shouldn't use this
+        variant.
+
+        With the keyword let. For example,</p>
+      <code>let y = 13.</code>
+      <p>This syntax can be used to declare a block scope local variable. See Variable scope below. </p>
+
+
+    </section>
+
+    <section id="Reference" class="main-section">
+      <header >
+        Reference
+      </header>
```

Comment:
- [ ]  Don't use header for individual headings of each part of the document. There should only be 1 header per page for semantic purposes. Instead, use a div or a section :smiley:

Reply:
- [ ] Hi, Please check my earlier comment about politeness. even when suggesting something that is crucial, please make it sound like a suggestion instead of a command 👍. It will read a lot better to the viewer.